cmd/plugin_test.go

```diff
@@ -475,6 +539,15 @@ func TestQuery(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("TestValidDirectRead", func(t *testing.T) {
+		transferStr := "pelican://something/here?directread"
```

Comment:
Similar to your other tests, the comments above this test says more than one query should fail. After you consolidate the two `checkValidQuery()`, you can delete the duplicate tests and move what's left into utils.